libpod/container.go

```diff
@@ -236,6 +236,7 @@ type ContainerConfig struct {
 	UserNsCtr   string `json:"userNsCtr,omitempty"`
 	UTSNsCtr    string `json:"utsNsCtr,omitempty"`
 	CgroupNsCtr string `json:"cgroupNsCtr,omitempty"`
+	NetNs       string `json:"netNs,omitempty"`
```

Comment:
Why do we need this? Why not create it in cmd/podman and pass it in the spec?

Reply:
I've not tried it but would it not get lost once we restart the container?

Reply:
Should be fine, we retain the whole spec and the path is in the spec?

Reply:
yes it is in the spec, but we delete the OCI container when we do a `podman restart`